### PR TITLE
Increase valkey-benchmark max latency bucket to 60 seconds

### PR DESCRIPTION
**Pull request by gabiganam**
## Summary

Increase the maximum latency histogram value in valkey-benchmark from 3 seconds to 60 seconds.

## Problem

When requests experience latencies greater than 3 seconds, they are capped at `CONFIG_LATENCY_HISTOGRAM_MAX_VALUE` (3,000,000 μsec) before being recorded in the HDR histogram. Due to HDR histogram bucket boundaries, these capped values are reported as ~3,000,319 μsec.

This causes two issues:

1. **Loss of precision**: All latencies above 3 seconds appear as the same value (~3,000,319 μsec), making it impossible to distinguish between a 4-second and a 30-second latency.

2. **Misleading metrics**: When benchmarking under stress conditions, slow queries, or network issues, the reported p99/p100 latencies significantly underrepresent the actual latency experienced.

### CODE REVIEW

**Comment by sarthakaggarwal97** [comment]
I think this is a good idea if the only outcome is that we will have more precision.

File: src/valkey-benchmark.c [hunk 1]
```diff
@@ -68,10 +68,10 @@
 #define MAX_LATENCY_PRECISION 4
 #define MAX_THREADS 500
 #define CLUSTER_SLOTS 16384
-#define CONFIG_LATENCY_HISTOGRAM_MIN_VALUE 10L              /* >= 10 usecs */
-#define CONFIG_LATENCY_HISTOGRAM_MAX_VALUE 3000000L         /* <= 3 secs(us precision) */
-#define CONFIG_LATENCY_HISTOGRAM_INSTANT_MAX_VALUE 3000000L /* <= 3 secs(us precision) */
-#define SHOW_THROUGHPUT_INTERVAL 250                        /* 250ms */
+#define CONFIG_LATENCY_HISTOGRAM_MIN_VALUE 10L               /* >= 10 usecs */
+#define CONFIG_LATENCY_HISTOGRAM_MAX_VALUE 60000000L         /* <= 60 secs(us precision) */
```
Comment (sarthakaggarwal97) [inline src/valkey-benchmark.c:72]: The PR description says requests upto 30s (which is also quite large). Do you see requests that can take upto 60s?

Comment (gabiganam) [reply src/valkey-benchmark.c:72]: For normal commands and/or in steady state probably not.
I'm using it to test commands that I expect to take a while (like scripts reading huge keys) and in edge-case states (like during load test or slot migrations).

Comment (sarthakaggarwal97) [reply src/valkey-benchmark.c:72]: Are there any side affects of increasing this limit? I am thinking what was the reason that it was kept at 3s for so long.

Comment (gabiganam) [reply src/valkey-benchmark.c:72]: I guess the "normal" use case of the benchmark tool is to test fast commands and reach high throughput, and my use case is not a classic usage and thus wasn't necessarily supported properly.
As far as side effects, the max value increase means there will be more buckets in the histogram, but I think it's negligible.

**Review by sarthakaggarwal97 — APPROVED** [review]

File: src/valkey-benchmark.c (hunk 1 above)
Comment (zuiderkwast) [reply src/valkey-benchmark.c:72]: How many more bucket? What's the size of the buckets? Power of two?

**Comment by zuiderkwast** [comment]
Majority decision is not required for benchmark tool changes. We've decided this earlier and done many changes without it.

**Review by sarthakaggarwal97 — APPROVED** [review]

File: src/valkey-benchmark.c (hunk 1 above)
Comment (sarthakaggarwal97) [reply src/valkey-benchmark.c]: They are powers of two. This adds 4 buckets, going from 9 to 13. That costs 32 KB per histogram. We use two histograms normally, so the total increase is 64 KB. With 16 threads, it’s about 1 MB. I think that’s fine for benchmarks.
